messages/en.json

```
@@ -3,5 +3,6 @@
   "store/app-store-components-agree-terms": "By placing this order, you are agreeing with the {appTermsAndConditions}",
   "store/app-store-components-terms-and-conditions": " {appName} terms and conditions.",
   "store/app-store-components-install-page-button": "Go to install page",
-  "store/app-store-components-confirmation-message": "Thank you for your purchase!{newLine}To complete the installation process on the {account} account, use the button below"
+  "store/app-store-components-confirmation-message": "Thank you for your purchase!{newLine}To complete the installation process on the {account} account, use the button below",
+  "store/app-store-components-billing-notice-message": "You will be charged a $0.01 fee in order to validate your credit card information. This will be refunded as soon as this payment is approved"
```

A moeda não está atrelada ao idioma da mensagem, provavelmente teria que receber por parâmetro ou emitir (e possivelmente exibir em outro lugar)

```suggestion
  "store/app-store-components-billing-notice-message": "You will be charged a {currency} 0.01 fee in order to validate your credit card information. This will be refunded as soon as this payment is approved"
```